### PR TITLE
Disallow invoking Run-ALPipeline with keepContainer without specifying credential

### PR DESCRIPTION
**Pull request by mazhelez**
If `keepContainer` is specified, `credential` must also be specified, as otherwise the container will be created with a random password and there will be no way to access it.

### CODE REVIEW

**Comment by freddydk** [comment]
Please also remove lines 670 - 678 (the dump on whether credentials is specified) - there is no reason to dump this info anymore.

File: AppHandling/Run-AlPipeline.ps1 [hunk 1]
```diff
@@ -485,6 +485,11 @@ if ($sharedFolder -and !(Test-Path $sharedFolder -PathType Container)) {
     throw "If sharedFolder is specified, it must be an existing folder"
 }
 
+if($keepContainer -and !$credential) {
```
Comment (freddydk) [inline AppHandling/Run-AlPipeline.ps1:488]: Please also remove lines 670 - 678 (the dump on whether credentials is specified) - there is no reason to dump this info anymore.

**Review by freddydk — CHANGES_REQUESTED** [review]

**Review by freddydk — CHANGES_REQUESTED** [review]
Please also remove lines 670 - 678 (the dump on whether credentials is specified) - there is no reason to dump this info anymore.

File: AppHandling/Run-AlPipeline.ps1 (hunk 1 above)
Comment (mazhelez) [reply AppHandling/Run-AlPipeline.ps1:488]: Done.

**Review by freddydk — APPROVED** [review]

**Pull request merged**